README.md

```diff
@@ -32,6 +32,30 @@ import { EIP191Signer } from '@lukso/eip191-signer.js';
 const eip191Signer = new EIP191Signer();
 ```
 
+### Example of Signing an LSP6KeyManager relay call
+
+The [LSP6-KeyManager](https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager#relay-execution) standard uses version 0 for signed messages. Therefore, it should us [`signDataWithIntendedValidator`](#signdatawithintendedvalidator).
+
+```js
```

Comment:
I would definitely suggest putting an example for ether.js as well, as ether.js is more used now compared to web3js.